ui/src/app/views/admin-user/component.html

```diff
@@ -9,13 +10,21 @@ <h2>Review ID Verifications</h2>
     <clr-dg-placeholder>No rejected or unverified profiles found.</clr-dg-placeholder>
     <clr-dg-row *clrDgItems="let profile of profiles" class="table-row workspace-table-row">
       <clr-dg-cell>{{profile.idVerificationStatus | lowercase}}</clr-dg-cell>
+      <clr-dg-cell>{{profile.disabled}}</clr-dg-cell>
       <clr-dg-cell>{{profile.familyName}}, {{profile.givenName}}</clr-dg-cell>
       <clr-dg-cell>{{profile.username}}</clr-dg-cell>
       <clr-dg-cell>{{profile.contactEmail}}</clr-dg-cell>
       <clr-dg-cell>{{profile.idVerificationRequestTime | date: 'medium'}}</clr-dg-cell>
       <clr-dg-cell>
-        <button class="btn" (click)="setIdVerificationStatus(profile,'VERIFIED')">Approve</button>
-        <button class="btn" (click)="setIdVerificationStatus(profile,'REJECTED')">Reject</button>
+        <div *ngIf="profile.idVerificationStatus !== IdVerificationStatus.VERIFIED">
+          <button class="btn"
+                  (click)="setIdVerificationStatus(profile,'VERIFIED')">Approve</button>
+          <button class="btn" (click)="setIdVerificationStatus(profile,'REJECTED')">Reject</button>
+        </div>
+        <div *ngIf="profile.idVerificationStatus === IdVerificationStatus.VERIFIED">
```

Comment:
Thinking aloud on how we could handle this if / when the user-disable is a separate authority... do we currently have client-side access to the authority list for the current user? If not, that might be a nice next step to increase the flexibility of this page.

Reply:
No, and I don't really want to expose authorities clientside if we can avoid it? It might make sense, but something about it worries me. I will think through that while doing the refactor.

Reply:
If we move towards a single "administer users" authority then I agree it won't be necessary... but if things develop in a way where "devs" should be able to perform more advanced administration than "support team" for some reason, we may end up wanting to pass this down to the client for a better user experience.

I wonder what the concerns would be? If all backend API calls are already protected separately, the only risk would be that someone could discover what actions are available to super-admins if they (a) guess an authority name and (b) somehow monkey-patch it into their client.

Maybe worthy of a tech talk topic?

Reply:
Sure! Lets throw it in a tech talk.